src/serde_multipart/mod.rs

```diff
@@ -31,6 +31,7 @@ pub(crate) fn to_form<T: ?Sized + Serialize>(val: &T) -> impl Future<Output = Re
 }
 
 // https://github.com/teloxide/teloxide/issues/473
+#[cfg(test)]
 #[tokio::test]
```

Comment:
Why is this necessary? `#[tokio::test]` [expands to `#[test]`](https://github.com/tokio-rs/tokio/blob/12dd06336d2af8c2d735d4d9e3dc0454ad7942a0/tokio-macros/src/entry.rs#L333) which in turn [applies `cfg(test)`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=931688976a0e166b3f5fc6d2499138a1).

Reply:
Weird. I can't reproduce the error, but this is how it looked before (copy-pasted from terminal, true footage):
```
   Compiling teloxide-core v0.3.4 (/home/aka_dude/Work/rust/notify-tg/teloxide-core)
error[E0433]: failed to resolve: could not find `test` in `tokio`
  --> teloxide-core/src/serde_multipart/mod.rs:34:10
   |
34 | #[tokio::test]
   |          ^^^^ could not find `test` in `tokio`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `teloxide-core` due to previous error
```

Reply:
Oh, that actually makes sense. We only enable tokio macros in `dev` builds, so in other builds the macro can't expand to anything since it doesn't exist
https://github.com/teloxide/teloxide-core/blob/5ccbf8b7b066a1bfde956bfb9d4a6581cd3b5550/Cargo.toml#L49-L51

Reply:
Weird how it works, and compiles most of the time.